lib/senders/templates/verify_secondary.txt

```diff
@@ -0,0 +1,13 @@
+{{t "Verify secondary email"}}
+{{t "A request to use %(email)s as a secondary email address has been made from the following Firefox Account:"}}
+{{#if primaryEmail }}{{ primaryEmail }}{{/if}}
+{{ device }}
+{{#if location}}{{ location }}{{/if}}
+{{#if ip}}{{t "IP address: %(ip)s" }}{{/if}}
+{{t "Verify email"}} {{{link}}}
```

Comment:
in the sign up email this is `Activate now: http://127.0.0.1:3030/verify_email?....`

Should we add a `:` here as well?